configure.cmake

```diff
@@ -556,6 +556,10 @@ IF(WITH_PERCONA_TELEMETRY)
   SET(HAVE_PERCONA_TELEMETRY 1)
 ENDIF()
 
+IF(WITH_PERCONA_PROBUILD OR PRO_BUILD)
+  SET(HAVE_PERCONA_PROBUILD 1)
+ENDIF()
+
```

Comment:
this is waht I did for pxb

+OPTION(PROBUILD "Build Percona XtraBackup Pro variant" OFF)
+IF (PROBUILD)
+ SET(XTRABACKUP_SUFFIX "-pro")
+ ADD_DEFINITIONS(-DPROBUILD)
+ENDIF()

Reply:
Also I used the cmake option PROBUILD. Better to use same name across all products. 

Or if you feel -DPERCONA_PROBUILD=1 is better, I will change it in next release.

@percona-ysorokin your thoughts?

Reply:
@satya-bodapati I followed `WITH_VALGRIND` and `WITH_PERCONA_TELEMETRY` style.
I also added `PROBUILD` option (in Jira there was `PRO_BUILD` but probably you changed it later) which is equal to `WITH_PERCONA_PROBUILD`.